documents/motions/10.2022.4 - Ethics Committee.md

```diff
@@ -1,35 +1,30 @@
-   5. Informing the WCA Board on investigations, decisions, and disciplinary actions.
-   6. Publishing public information on the investigations, decisions, and disciplinary actions where necessary.
-   7. Managing an administration of the investigations, the reports, the decisions, and the disciplinary actions.
-   8. Being a confidant to WCA Staff and Registered Speedcubers regarding ethical matters and questions within the WCA.
-   9. WCA Staff and Registered Speedcubers must never start communications with the WCA Ethics Committee regarding matters related to the WCA Disciplinary Committee or the WCA Ethics Committee, except for the following matters:
-      1. Requests for investigations by the WCA Board and the WCA Disciplinary Committee.
-      2. Requests for appeal by the WCA Board on behalf of a Registered Speedcuber or WCA Staff.
-      3. Financial Support of the WCA Ethics Committee by the WCA Financial Committee.
-      4. Questions and reports to the confidant within the WCA Ethics Committee.
-2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Ethics Committee.
-3. In case the WCA Ethics Committee is not constituted or for any reason non-functional, the WCA Disciplinary Committee will act as WCA Ethics Committee.
-4. The WCA Ethics Committee will take the role of WCA Disciplinary Committee in case the WCA Disciplinary Committee is not constituted or for any reason non-functional.
-5. Reducing conflicts of roles
-   1. There may be no overlap in members of the WCA Board, WCA Disciplinary Committee, and WCA Ethics Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader.
+1. The WCA Appeals Committee has the following rights and duties:
+   1. Forming the Board of Appeal to carry out the appeals processes described in [WCA Motion : Disputes](wca{documents/motions/16.2021.1 - Disputes.pdf}).
+      1. The Board of appeal shall be made up of three members of the WCA Appeals Committee
+   2. Performing independent investigations regarding appealed incidents
+   3. Making decisions on disciplinary actions in relation to appealed incidents.
+   4. Informing the WCA Board on appeals, investigations, decisions, and disciplinary actions.
+   5. Publishing public information on the appeals, investigations, decisions, and disciplinary actions where necessary.
+   6. Managing an administration of the appeals, investigations, the reports, the decisions, and the disciplinary actions.
+2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Appeals Committee.
+3. In case the WCA Appeals Committee is not constituted or for any reason non-functional, the WCA Board will act as WCA Appeals Committee.
+4. The WCA Appeals Committee will take the role of WCA Integrity Committee in case the WCA Integrity Committee is not constituted or for any reason non-functional.
```

Comment:
add that the appeals committee also acts as the WIC when investigating the actions of WIC members?

Reply:
isn't it covered by the line below? "For any period where the WCA Appeals Committee assumes the role of the WCA Integrity Committee the WCA Board may appoint select WCA Staff to handle cases of appeal."